projects/plugins/jetpack/tests/php/sync/Jetpack_Sync_Post_Test.php

```diff
@@ -1457,15 +1451,9 @@ public function test_sync_jetpack_posts_akismet_post_meta_delete_is_chunked() {
 	public function test_sync_jetpack_posts_akismet_post_meta_delete_invalid_data() {
 		$ids = 'test_invalid_value';
 
-		$mocked = $this->getMockBuilder( stdClass::class )
-						->addMethods( array( 'chunked_call' ) )
-						->getMock();
-
-		$mocked->expects( $this->never() )
-				->method( 'chunked_call' );
-
-		// @phan-suppress-next-line PhanEmptyFQSENInClasslike -- https://github.com/phan/phan/issues/4851
-		add_action( 'jetpack_post_meta_batch_delete', array( $mocked, 'chunked_call' ), 10, 2 );
+		$callback = $this->getMockBuilder( \CallableMock::class )->getMock();
+		$callback->expects( $this->never() )->method( '__invoke' );
```

Comment:
Is there any difference between `exactly(0)` and `never()`?

Reply:
I don't think so.